tests/parametric/test_otel_env_vars.py

```diff
@@ -85,6 +92,7 @@ def test_otel_env_vars_set(self, test_agent, test_library):
     @missing_feature(context.library == "python", reason="DD_LOG_LEVEL is not supported in Python")
     @missing_feature(context.library == "dotnet", reason="DD_LOG_LEVEL is not supported in .NET")
     @missing_feature(context.library == "ruby", reason="DD_LOG_LEVEL is not supported in ruby")
+    @missing_feature(context.library == "java", reason="OTEL_LOG_LEVEL doesn't update DD_LOG_LEVEL")
```

Comment:
Note that both `OTEL_LOG_LEVEL` and `DD_LOG_LEVEL` control the tracer's built-in logger, but this is done well before the configuration system is up, in a separate file: https://github.com/DataDog/dd-trace-java/blob/v1.39.0/dd-java-agent/agent-bootstrap/src/main/java/datadog/trace/bootstrap/Agent.java#L1114

That's why there's no later mapping for `Config`, as it has already been mapped and the tracer's logger doesn't read `Config`  - you could always use `System.getProperty("datadog.slf4j.simpleLogger.defaultLogLevel")` in `getTraceConfig` to get the tracer's configured log-level, which would let you un-ignore this test.